res/css/views/elements/_AccessibleButton.scss

```diff
@@ -137,15 +137,11 @@ limitations under the License.
         font-size: inherit;
         font-weight: normal;
         line-height: inherit;
-    }
-
-    &.mx_AccessibleButton_kind_link {
         padding: 0;
     }
 
     &.mx_AccessibleButton_kind_link_inline {
         display: inline;
-        padding: 0 2px;
```

Comment:
There is a lot of declarations to cancel the inline 2px padding, and it is not quite sure what this has been introduced for.